tests/integration_tests/config/generate_default_metadata_to_yml_file.yml

```diff
@@ -1,6 +1,9 @@
 CTA_SIMPIPE:
   APPLICATION: simtools-generate-default-metadata
-  TEST_NAME: test
+  TEST_NAME: test-yaml
```

Comment:
That made realized that the same issue with the json test is present here (missing --schema argument). I would suggest including it here too.

Reply:
see previous comment.